fcrepo-kernel-modeshape/src/test/java/org/fcrepo/kernel/modeshape/rdf/converters/PropertyConverterTest.java

```diff
@@ -116,6 +116,14 @@ public final void shouldRejectFcrPredicates() throws RepositoryException {
         PropertyConverter.getPropertyNameFromPredicate(mockNode, p, nsMap);
     }
 
+    @Test(expected = RuntimeException.class)
+    public final void IllegalPredicateLocalname() throws RepositoryException {
+        final Property p = createProperty(mockUri, "1234");
+        final Map<String, String> nsMap = new HashMap<String, String>();
```

Comment:
Just `final Map<String, String> nsMap = ImmutableMap.of("example", mockUri)`, and maybe `static import` the `of`, or inline the whole value.